joomla/joomla-cms
# [4.3] Fixing Itemid lookup in tags router in multilang

Pull Request for Issue #40454.

### Summary of Changes
When running in multilanguage mode, the router falsely only checked for the current language or the default language, not for all (`*`) language when looking up the right Itemid. This executes the logic twice, once for the current language, once for the `*` language. It exits for the first matched menu item.


### Testing Instructions
1. Install Joomla
2. Install at least one additional language
3. Install multilanguage sample data
4. Install testing sample data
5. Go to for example the popular tags module. You can find it in the "Popular tags" menu item.
6. Check the URLs


### Actual result BEFORE applying this Pull Request
URLs start with /component/tags/...


### Expected result AFTER applying this Pull Request
URLs have different menu items. None start with /component/tags


### Link to documentations
Please select:
